yarn.lock

```diff
@@ -16346,6 +16356,39 @@ react-native-webview@11.2.3:
     escape-string-regexp "2.0.0"
     invariant "2.2.4"
 
+react-native@0.63.2:
```

Comment:
I'm not sure react-native should ever show up in yarn.lock -- historically this has indicated issues with not using the symlinked copy from the submodule. You may need to run git submodule init, etc...